web/packages/teleport/src/Console/DocumentNodes/useNodes.ts

```diff
@@ -103,3 +102,12 @@ export default function useNodes({ clusterId, id }: stores.DocumentNodes) {
     ...paginationProps,
   };
 }
+
+// sort logins by making 'root' as the first in the list
+export const sortLogins = (logins: string[]) => {
+  const noRoot = logins.filter(l => l !== 'root').sort();
+  if (noRoot.length === logins.length) {
+    return logins;
+  }
+  return ['root', ...noRoot];
+};
```

Comment:
```suggestion
export const sortLogins = (logins: string[]): string[] => {
  return [...logins].sort((a, b) => {
    if (a === 'root') return -1;
    if (b === 'root') return 1;
    return a.localeCompare(b);
  });
};
```

you dont have to update this, but i can be done this way as well I think. to me, its a bit more readable but idc either way

Reply:
@avatus I was thinking the same, but have an opposite opinion xD

Reply:
i kept it the same, except applied @bl-nero comment about separately sorting